### ISSUE TITLE
per-host transaction log (add syslog support)

### ISSUE THREAD

**Issue by digint** [digint/btrbk#82]
btrbk writes a `transaction_log`, but it is not yet possible to have that separated.

#### Add support to override `transaction_log` in **config sections**

e.g.
```
volume ssh://alpha.example.com/mnt/btr_pool
  transaction_log /var/log/btrbk/alpha.btrbk.log

volume ssh://beta.example.com/mnt/btr_pool
  transaction_log /var/log/btrbk/beta.btrbk.log
```

#### Syslog support (analog to transaction log)

```
volume ssh://alpha.example.com/mnt/btr_pool
  syslog_priority <prio>
  syslog_server <server>
```

**Comment by digint** [comment]
syslog implemented in: e9e398af37db768c87d28de7bee66095b0737ec5
(thanks @knorrie)

**Comment by digint** [comment]
Syslog support is implemented in [btrbk-v0.23.1](/digint/btrbk/releases/tag/v0.23.1), with the `transaction_syslog` configuration option.

The idea of a per-host transaction-log was dropped, as this would make it very confusing for hosts defined in volume AND/OR target sections.
